### ISSUE TITLE
UAT for Courtesy Cards

### ISSUE THREAD

**Issue by MarikoLochridge** [cal-itp/benefits#1034]
Create UAT for courtesy cards. See sample templates in link below. 

https://docs.google.com/spreadsheets/d/1daAf6UzSuEENYpb-OeTjlaQZ5HizalgwO9wYDoYQAuE/edit#gid=740449822

**Comment by machikoyasuda** [comment]
- Define devices, browsers, a11y flows should be tested: Devices (Android, iPhone), Browsers (Chrome, Firefox, iOS Safari, Android Chrome), a11y flows (Desktop without mouse, Screenreader)
- Developers responsible for getting Product all the credentials needed: @machikoyasuda to update new login.gov yml credentials
- Developers ensure all screens covered: @machikoyasuda to ensure all screens covered
- Screenreaders – explore best software options for testing in windows and mac: @machikoyasuda to figure out screenreading on iOS and Mac

**Comment by MarikoLochridge** [comment]
Created a UAT form for Courtesy Cards
https://docs.google.com/spreadsheets/d/1WZdPBURZgPNXWndRkulKimab-S5aHASTacihuE3Qlrg/edit?usp=sharing

Tested on Macbook, iPhone, Windows and Android

**Comment by MarikoLochridge** [comment]
Courtesy Cards tested by @machikoyasuda @afeld @angela-tran 

Comments:
On Windows by @angela-tran 
- It'd be nice if the final success page had a way to go back home in case the user wants to add another card
- I also tested with toggling from English to Spanish

On Android by @angela-tran 
- I'm not sure how useful my test is since I'm on pretty old versions of Android / Chrome
- It'd be nice to not have to scroll down on the home page to get to the "Get Started" button

@MarikoLochridge 
I completed the valid user Courtesty Card flow for all 12 valid user information on the Benefits Test Data doc. I did a combination of copy/paste as well as direct input into the courtesy card flow prompts and was successfully connected each time. I received the error message only once when I incorrectly reversed the name and the courtesy card number.
